components/suggestion/at_mention_provider/at_mention_provider.jsx

```diff
@@ -99,6 +99,11 @@ export default class AtMentionProvider extends Provider {
         return groupSuggestions;
     }
 
+    // normalizeSuggestions removes accents from strings to match ascii encoding
+    normalizeSuggestions(name) {
+        return name.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
```

Comment:
I didn't know it was actually this easy to remove accents from strings. I thought we'd have to manually replace the characters. I imagine this

Reply:
Yeah! Way easier than the first approach I followed.